styles.css

```diff
@@ -102,3 +102,227 @@ header a {
   border-radius: 32px;
   padding: 8px;
 }
+
+.recent-projects {
+  margin-top: 22px;
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: repeat(7, auto);
+  justify-content: center;
+}
+
+.recent-work-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.recent-work-title h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.green-line {
+  margin-top: 24px;
+  width: 48px;
+  height: 4px;
+  background-color: #40bb82;
+  text-align: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.card1 {
+  margin-top: 62px;
+}
+
+.card2,
+.card3,
+.card4,
+.card5,
+.card6 {
+  margin-top: 24px;
+}
+
+.image-card-holder {
+  height: 252px;
+  background-color: #ebf0ee;
+}
+
+.text-work-title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  margin-top: 16px;
+}
+
+.card-tags {
+  margin-top: 16px;
+}
+
+.card-tags ul {
+  display: flex;
+  list-style: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  justify-content: space-around;
+}
+
+.tag-element {
+  padding: 8px 12px;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+}
+
+.tag-element:hover {
+  background-color: #36b37f;
+}
+
+.see-project-btn {
+  width: 122px;
```

Comment:
- [ ] I salute your work of art, you have accomplished a great work, there are a few things you will need to work on before you go to the next step, let's take a look at this below and make all as required, please 😄 
-  Kindly implement the button actions as `hover, disable, enable, active` as per the project requirement, see this image below ⏬ 

<img width="482" alt="actio" src="https://user-images.githubusercontent.com/79058364/159062435-4f064e9f-c46f-4ae7-af48-1f2cf290cd50.png">
 
- Don't you think this color does not match the original design? alright, please take a look closer at it and make the change 😄 
### YOUR DESIGN
<img width="309" alt="color" src="https://user-images.githubusercontent.com/79058364/159063055-8f93971b-1db9-45f6-b9f9-e174ee608f26.png">

###  FIGMA

<img width="243" alt="gik" src="https://user-images.githubusercontent.com/79058364/159063123-1240d17d-d086-4a81-98eb-d35eef9ea2f4.png">

- If you take a look on line 20 in your HTML, I think you left this `>` there and now it displays to your design 😞, please remove it. 
<img width="121" alt="png" src="https://user-images.githubusercontent.com/79058364/159063674-979ffc7c-960b-4a7b-9e17-ac34722f5d59.png">